HUD/CodeWantsScriptToTakeControl.md

```diff
@@ -0,0 +1,16 @@
+---
+ns: HUD
+aliases: ["0x66E7CB63C97B7D20"]
+---
+## CODE_WANTS_SCRIPT_TO_TAKE_CONTROL
+
+```c
+// 0x66E7CB63C97B7D20 0x92DAFA78
+cs_type(Any) BOOL CODE_WANTS_SCRIPT_TO_TAKE_CONTROL();
+```
+
+
+## Return value
+Returns true if game code wants script to take control of a Pause Menu screen this frame.
+Use [`GET_SCREEN_CODE_WANTS_SCRIPT_TO_CONTROL`](#_0x593FEAE1F73392D4) to get value of which screen should be controlled.
```

Comment:
This should be on native description and not on return value.

Description could be `Continuously monitor this condition every frame; if it returns true, verify CommandGetScreenCodeWantsScriptToTakeControl. Use [`GET_SCREEN_CODE_WANTS_SCRIPT_TO_CONTROL`](#_0x593FEAE1F73392D4) to get value of which screen should be controlled.`